clang/test/CodeGen/builtin-assume-dereferenceable.c

```diff
@@ -32,3 +32,62 @@ int test2(int *a) {
   __builtin_assume_dereferenceable(a, 32ull);
   return a[0];
 }
+
+// CHECK-LABEL: @test3(
+// CHECK-NEXT:  entry:
+// CHECK-NEXT:    [[A_ADDR:%.*]] = alloca ptr, align 8
```

Comment:
Alignments in the tests make this a touch fragile, but I see the rest of the test does it, so *YOLO*.